db/schema_tables.cc

```diff
@@ -1697,6 +1697,13 @@ static std::vector<data_value> read_arg_values(const query::result_set_row& row)
     }
 #endif
 
+static std::pair<cql3::functions::function_name, std::vector<data_type>> drop_func(replica::database& db, const query::result_set_row& row) {
+    cql3::functions::function_name name{
+            row.get_nonnull<sstring>("keyspace_name"), row.get_nonnull<sstring>("function_name")};
+    auto arg_types = read_arg_types(db, row, name.keyspace);
+    return std::make_pair(std::move(name), std::move(arg_types));
+}
+
 static shared_ptr<cql3::functions::user_function> create_func(replica::database& db, const query::result_set_row& row) {
```

Comment:
This one actually does create something. So better rename drop_func() to reflect what it does.

Reply:
I've changed it so the drop function is actually added in the same patch as the cache, and it does what is says